libs/dal/src/repositories/subscriber/subscriber.schema.ts

```diff
@@ -25,7 +25,7 @@ const subscriberSchema = new Schema(
     channels: [Schema.Types.Mixed],
     isOnline: {
       type: Schema.Types.Boolean,
-      default: false,
+      required: false,
```

Comment:
changed the field `isOnline` to not required, otherwise it sets `isOnline: false` which is not correct, because the subscriber might not be offline